ush/config.sh.RRFS_dev1

```diff
@@ -36,23 +36,29 @@ QUILTING="TRUE"
 CCPP_PHYS_SUITE="FV3_HRRR"
 
 EXTRN_MDL_ICS_OFFSET_HRS="0"
-BOUNDARY_LEN_HRS="15"
-BOUNDARY_LONG_LEN_HRS="36"
+BOUNDARY_LEN_HRS="21"
+BOUNDARY_LONG_LEN_HRS="51"
 LBC_SPEC_INTVL_HRS="1"
 EXTRN_MDL_LBCS_OFFSET_HRS="0"
 EXTRN_MDL_LBCS_SEARCH_OFFSET_HRS="0"
 
-INITIAL_CYCLEDEF="00 09,21 * * 2021 *"
-BOUNDARY_CYCLEDEF="00 00-02/01,04-08/01,10-14/01,16-20/01,22,23 * * 2021 *"
-BOUNDARY_LONG_CYCLEDEF="00 03,09,15,21 * * 2021 *"
-PREP_COLDSTART_CYCLEDEF="00 09,21 * * 2021 *"
-PREP_WARMSTART_CYCLEDEF="00 00-08/01,10-20/01,22-23/01 * * 2021 *"
-ANALYSIS_CYCLEDEF="00 00-23/01 * * 2021 *"
-FORECAST_CYCLEDEF="00 00-23/01 * * 2021 *"
-ARCHIVE_CYCLEDEF="00 07 * * 2021 *"
-
-FCST_LEN_HRS="24"
-FCST_LEN_HRS_CYCLES=(24 3 3 12 3 3 24 3 3 12 3 3 24 3 3 12 3 3 24 3 3 12 3 3)
+#CYCLEDAY="1-4"
+CYCLEMONTH="5-7"
+INITIAL_CYCLEDEF="00 09,21 ${CYCLEDAY} ${CYCLEMONTH} 2021 *"
+BOUNDARY_CYCLEDEF="00 00-02/01,04-08/01,10-14/01,16-20/01,22,23 ${CYCLEDAY} ${CYCLEMONTH} 2021 *"
+BOUNDARY_LONG_CYCLEDEF="00 03,09,15,21 ${CYCLEDAY} ${CYCLEMONTH} 2021 *"
+PREP_COLDSTART_CYCLEDEF="00 09,21 ${CYCLEDAY} ${CYCLEMONTH} 2021 *"
+PREP_WARMSTART_CYCLEDEF="00 00-08/01,10-20/01,22-23/01 ${CYCLEDAY} ${CYCLEMONTH} 2021 *"
+ANALYSIS_CYCLEDEF="00 00-23/01 ${CYCLEDAY} ${CYCLEMONTH} 2021 *"
+FORECAST_CYCLEDEF="00 00-23/01 ${CYCLEDAY} ${CYCLEMONTH} 2021 *"
+POSTPROC_CYCLEDEF="00 01-05,07-11,13-17,19-23 ${CYCLEDAY} ${CYCLEMONTH} 2021 *"
+POSTPROC_LONG_CYCLEDEF="00 00,06,12,18 ${CYCLEDAY} ${CYCLEMONTH} 2021 *"
+ARCHIVE_CYCLEDEF="00 07 ${CYCLEDAY} ${CYCLEMONTH} 2021 *"
+
+FCST_LEN_HRS="18"
+POSTPROC_LEN_HRS="18"
+POSTPROC_LONG_LEN_HRS="48"
+FCST_LEN_HRS_CYCLES=(48 18 18 18 18 18 48 18 18 18 18 18 48 18 18 18 18 18 48 18 18 18 18 18)
```

Comment:
It is easy to mis-specify values for FCST_LEN_HRS_CYCLES, we may change it to:
```
for i in {0..23}; do FCST_LEN_HRS_CYCLES[$i]=18; done
for i in {0..23..3}; do FCST_LEN_HRS_CYCLES[$i]=48; done
```
The first line sets all cycles to 18h fcst length first and the second line changes 3hr-ly cycles to 48h fcst length

Reply:
Good idea. We can use this way after I try it.